wesnoth-multiplayer.po

```diff
@@ -1425,6 +1418,13 @@ msgid ""
 "eternal scar on the land that to this day summons forth legions of wandering "
 "undead."
 msgstr ""
+"Kdysi bylo město Myrilel klidnou vesnicí na nejvýchodnějším cípu lidského "
+"osídlení. To se navždy změnilo po vzestupu válečného lorda licha Mal-"
+"Ravanala, který rozerval centrum města branou stvořenou nevyslovitelně "
+"temným čarodějnictvím. Vytvořil tak první místo pro shromažďování svých "
+"armád smrti. Ačkoli byl Mal-Ravanal poražen, Myrilel zůstalo zpustošené a "
+"zničené. Brána zůstává věčnou jizvou na zemi, co dodnes vyvrhává legie "
```

Comment:
to vaše co (použil bych která, ale na tom nesejde) se vztahuje k zemi, ale patří k bráně nebo jizvě

proto následující návrh
```suggestion
"zničené. Brána zůstává věčnou jizvou na zemi a dodnes vyvrhává legie "
```